src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -1,43 +1,26 @@
-import { createTxNotifications } from 'src/logic/notifications'
-import {
-  getApprovalTransaction,
-  getExecutionTransaction,
-  saveTxToHistory,
-  tryOffChainSigning,
-} from 'src/logic/safe/transactions'
-import { estimateSafeTxGas, SafeTxGasEstimationProps, createSendParams } from 'src/logic/safe/transactions/gas'
-import { currentSafeCurrentVersion } from 'src/logic/safe/store/selectors'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { getPreValidatedSignatures } from 'src/logic/safe/safeTxSigner'
 import { ZERO_ADDRESS } from 'src/logic/wallets/ethAddresses'
 import { EMPTY_DATA } from 'src/logic/wallets/ethTransactions'
-import { providerSelector } from 'src/logic/wallets/store/selectors'
-import { generateSafeTxHash } from 'src/logic/safe/store/actions/transactions/utils/transactionHelpers'
-import { getNonce, canExecuteCreatedTx } from 'src/logic/safe/store/actions/utils'
-import fetchTransactions from './transactions/fetchTransactions'
-import { AppReduxState } from 'src/store'
-import { Dispatch, DispatchReturn } from './types'
-import { checkIfOffChainSignatureIsPossible, getPreValidatedSignatures } from 'src/logic/safe/safeTxSigner'
 import { TxParameters } from 'src/routes/safe/container/hooks/useTransactionParameters'
-import { Errors, logError } from 'src/logic/exceptions/CodedException'
-import { extractShortChainName, history, SAFE_ROUTES } from 'src/routes/routes'
-import { getPrefixedSafeAddressSlug, SAFE_ADDRESS_SLUG, TRANSACTION_ID_SLUG } from 'src/routes/routes'
-import { generatePath } from 'react-router-dom'
-import { fetchOnchainError } from 'src/logic/contracts/safeContractErrors'
-import { isMultiSigExecutionDetails } from '../models/types/gateway.d'
-import { removePendingTransaction, addPendingTransaction } from 'src/logic/safe/store/actions/pendingTransactions'
-import { _getChainId } from 'src/config'
-import { GnosisSafe } from 'src/types/contracts/gnosis_safe.d'
-import * as aboutToExecuteTx from 'src/logic/safe/utils/aboutToExecuteTx'
-import { getLastTransaction } from '../selectors/gatewayTransactions'
-import { TxArgs } from '../models/types/transaction'
+import { AppReduxState } from 'src/store'
+import { TxArgs } from 'src/logic/safe/store/models/types/transaction'
+import { getLastTransaction } from 'src/logic/safe/store/selectors/gatewayTransactions'
+import { Dispatch, DispatchReturn } from 'src/logic/safe/store/actions/types'
+import { canExecuteCreatedTx, getNonce } from 'src/logic/safe/store/actions/utils'
+import { generateSafeTxHash } from 'src/logic/safe/store/actions/transactions/utils/transactionHelpers'
+import { GnosisSafe } from 'src/types/contracts/gnosis_safe'
+import { getGnosisSafeInstanceAt } from 'src/logic/contracts/safeContracts'
+import { providerSelector } from 'src/logic/wallets/store/selectors'
+import { currentSafeCurrentVersion } from '../selectors'
+import { ConfirmEventHandler, ErrorEventHandler, TxSender } from './TxSender'
```

Comment:
I am getting a warning because the filename is lowercase and should be renamed to `TxSender.ts`

Reply:
Guess we'll have to stick with the lowercase name for now.